test_metannotate_end_to_end.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+"""
+Created by: Metannotate Team (2017)
+
+Description: A shell script that runs a sample test of Metannotate and compares the generated
+             output files to verify proper end-to-end functionality of the program.
+
+Requirements: - The following packages:
+                    - build-essential
+                    - python-dev
+                    - curl
+                    - file
+                    - git
+                    - python-setuptools
+                    - ruby
+              - Base installation (sudo -H bash base_installation.sh) of Metannotate
```

Comment:
@KyleBehiels BTW we can't use sudo with this install script with ```sudo -H``` as this will literally break the script since hombrew is designed to work without sudo and will actively give an error and exit if it is called through sudo. It is generally accepted that you should never run install scripts without sudo. If we need sudo we can put it inside the install script (as in call sudo in front of the command inside the install script). I have seen many problems with people just blinding calling sudo in front of things. Often it installs things under root making them in accessible to the actual user (the root writes the files so the end user does not have read write permissions). Always call such scripts first without sudo and only use sudo when you have to.